### ISSUE TITLE
当我启用sage_atten的时候，依然返回的是使用flash attention的信息？

### ISSUE THREAD

**Issue by swan7-py** [Tencent-Hunyuan/HunyuanVideo-1.5#9]
我运行下面的脚本

> export PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True,max_split_size_mb:128
> export T2V_REWRITE_BASE_URL=""
> export T2V_REWRITE_MODEL_NAME=""
> export I2V_REWRITE_BASE_URL=""
> export I2V_REWRITE_MODEL_NAME=""
> 
> PROMPT='A girl holding a paper with words "Hello, world!"'
> 
> IMAGE_PATH=none # Optional, 'none' or <image path>
> SEED=1
> ASPECT_RATIO=16:9
> RESOLUTION=720p
> OUTPUT_PATH=./outputs/output.mp4
> 
> #Configuration
> N_INFERENCE_GPU=1 # Parallel inference GPU count
> CFG_DISTILLED=false # Inference with CFG distilled model, 2x speedup
> SPARSE_ATTN=false # Inference with sparse attention (only 720p models are equipped with sparse attention). Please ensure flex-block-attn is installed
> SAGE_ATTN=True # Inference with SageAttention
> REWRITE=False # Enable prompt rewriting. Please ensure rewrite vLLM server is deployed and configured.
> OVERLAP_GROUP_OFFLOADING=False # Only valid when group offloading is enabled, significantly increases CPU memory usage but speeds up inference
> MODEL_PATH=ckpts # Path to pretrained model
> 
> torchrun --nproc_per_node=$N_INFERENCE_GPU generate.py \
>   --prompt "$PROMPT" \
>   --image_path $IMAGE_PATH \
>   --resolution $RESOLUTION \
>   --aspect_ratio $ASPECT_RATIO \
>   --seed $SEED \
>   --cfg_distilled $CFG_DISTILLED \
>   --sparse_attn $SPARSE_ATTN \
>   --use_sageattn $SAGE_ATTN \
>   --rewrite $REWRITE \
>   --output_path $OUTPUT_PATH \
>   --overlap_group_offloading $OVERLAP_GROUP_OFFLOADING \
>   --save_pre_sr_video \
>   --model_path $MODEL_PATH

之后返回一个信息如下：

> Rewritten Prompt:          <disabled>
> Aspect Ratio:              16:9
> Video Length:              121
> Reference Image:           None
> Guidance Scale:            6.0
> Guidance Embedded Scale:   None
> Shift:                     9.0
> Seed:                      1
> Video Resolution:          1280 x 720
> Attn mode:                 flash
> Transformer dtype:         torch.bfloat16
> Sampling Steps:            50
> Use Meanflow:              False

在attn mode的位置依然是显示flash，这是为什么？

**Comment by KimbingNg** [comment]
The log is incorrect for now and can be ignored. You are actually using sage attention if you have set `--use_sageattn true`. We will update the logging soon
